Form/FormMapper.php

```diff
@@ -301,4 +301,12 @@ protected function setTabs(array $tabs)
     {
         $this->admin->setFormTabs($tabs);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName()
+    {
+        return 'form';
```

Comment:
Hum are you sure about this ? This is only called here : https://github.com/sonata-project/SonataAdminBundle/pull/4028/files#diff-d561f27bd57cd565613d211d8d8bb0afR73 and this is not related to symfony ``AbstractType``